exec.go

```diff
@@ -28,7 +28,7 @@ func NewEnviron(dir string, env []string, stdout, stderr io.Writer) *Environ {
 
 // Run executes the given program.
 func (e *Environ) Run(name string, arg ...string) error {
-	displayArg := reRedact.ReplaceAll([]byte(strings.Trim(fmt.Sprint(arg), "[]")), []byte("$1 [redacted] "))
+	displayArg := reRedact.ReplaceAll([]byte(strings.Trim(fmt.Sprint(arg), "[]")), []byte(" $1 [redacted] "))
```

Comment:
You can use `ReplaceAllString` here to avoid the type casting.

Reply:
+1

Reply:
cool, fixed, thanks.

new test https://drone-general-dev.newsdev.net/nytm/jryan-test/27

```
Drone GAE Plugin built from aceada7
...
Running Command: /go_appengine/appcfg.py  --oauth2_access_token  [redacted] 
-A nyt-tinctest1-dev -V 8807f08bbf0d24ba21900179a9063e8bef2efd18 -E myothervar: [redacted]  -E myvar: [redacted]  update .
```